content/en/docs/Getting started/install.md

```diff
@@ -106,12 +106,13 @@ For instance, it is possible to install cOS (or any derivative) with the install
 If in the rescue system, or LiveCD you have docker available, it can be used to perform an installation
 
 ```bash
-docker run --privileged -v $DEVICE:$DEVICE -ti quay.io/costoolkit/releases-green:cos-system-0.7.4-2 elemental install --docker-image $IMAGE $DEVICE
+docker run --privileged -v /dev/:/dev/ -ti quay.io/costoolkit/elemental:v0.0.14 install --docker-image $IMAGE $DEVICE
 ```
 
-Where `$IMAGE` is the container image that we want to install (e.g. `quay.io/costoolkit/releases-green:cos-system-0.7.4-2` ), and `$DEVICE` is the the device where to perform the installation to (e.g. /dev/sda).
+Where `$IMAGE` is the container image that we want to install (e.g. `quay.io/costoolkit/releases-green:cos-system-0.7.4-2` ), and `$DEVICE` is the device where to perform the installation to (e.g. /dev/sda).
```

Comment:
I would no use anything below cos-system-0.8 on docs, I think `v0.7.x` does not include elemental

Reply:
Good point, Im gonna point to latest image available instead